packages/toolpad-app/src/toolpad/Home/index.tsx

```diff
@@ -417,22 +417,24 @@ function AppSettingsDialog({ app, open, onClose }: AppSettingsDialogProps) {
       <DialogForm onSubmit={doSubmit}>
         <DialogTitle>Application settings for &quot;{app.name}&quot;</DialogTitle>
         <DialogContent>
-          <FormControlLabel
-            control={
-              <Controller
-                control={control}
-                name="public"
-                render={({ field: { value, onChange, ...field } }) => (
-                  <Checkbox
-                    checked={value}
-                    onChange={(e) => onChange(e.target.checked)}
-                    {...field}
-                  />
-                )}
-              />
-            }
-            label="Make application public"
-          />
+          {!config.isDemo ? (
```

Comment:
Should we disable the button with a message to avoid creating an empty dialog?

Reply:
Good thinking, this shouldn't show anyway for now as we don't show the apps screen in the demo, but I think it's fine to have it just in case.
<img width="647" alt="Screen Shot 2022-10-26 at 20 03 51" src="https://user-images.githubusercontent.com/10789765/198114538-5d265b9b-1ee1-4d1e-a1a2-eee56b6c52cc.png">

Reply:
I think @bharatkashyap  was working on bringing the menu to the app editor as well